server/routes/incidentDetails/incidentDetailsSubmittedEdit.ts

```diff
@@ -51,13 +52,18 @@ export default class IncidentDetailsSubmittedEditRoutes {
       locationId: location,
     }
 
+    const cancelButtonHref = (referrer as string)?.includes('review')
```

Comment:
Best practice is to avoid relying on a URL name (or any string that might change) to decide what to do next.

But there is no easy fix with the way things are right now.. I'll make a note